app/src/androidTest/java/com/sdp/movemeet/view/map/MainMapFragmentTest.java

```diff
@@ -63,41 +86,135 @@
 @RunWith(AndroidJUnit4.class)
 public class MainMapFragmentTest {
 
-    private UiDevice uiDevice;
+    private UiDevice uiDevice = UiDevice.getInstance(getInstrumentation());
```

Comment:
Why did you change this line to use a specific instance of the uiDevice instead of the uiDevice object itself?

Reply:
The UIDevice was used to, for instance, click on the map. This should launch a "onMapClick" method. But, since the map was never loaded by Cirrus, this method was not called. Instead, I directly call this method in my tests (with the condition that the map is not null), which makes the UIDevice useless, so I simply removed it.